cloudify_cli/commands/blueprints.py

```diff
@@ -299,7 +306,8 @@ def get(blueprint_id, logger, client, tenant_name):
     blueprint_deployments = [d['id'] for d in deployments]
 
     if get_global_json_output():
-        columns += ['description', 'metadata', 'deployments']
+        columns += ['description', 'metadata', 'deployments',
+                    'error_traceback']
```

Comment:
Why did you choose to add `error_traceback` and not `state`?

Reply:
state is already there